### PR TITLE
Changeling Balance#2

### PR DESCRIPTION
**Pull request by adacovsk**
Working on a few more changes that were brought up as feedback from the previous PR:
- Possibly incentivize lings killing eachother (?). Or less incentive for being friendly with eachother (?)
- Split removes a counter from absorbedcount
- dnapoints_req now triggers properly
- Added split icon
- Added cooldown for Regenerative Stasis

Feedback welcome on potential changes that I can implement as lings are now live again, as time goes by.

🆑 
 * tweak: Changeling: Split now costs DNA and borgs can't reproduce. Regenerative Stasis now has a 8 min cooldown. Fixed dnapoints_req, should trigger properly now. Fire multiplier reduced to 2x. Paralyzing sting now stuns.

### CODE REVIEW

**Comment by djkramer123** [comment]
>be changeling
>spend resources to split and make more changelings
>they're incentivized to kill me
I only vaguely read the first pr but is this right?

**Comment by djkramer123** [comment]
I only vaguely read the first PR but now there's only one ling when the ruleset fires, he spends resources to spawn another ling from ghosts which are then incentivized to kill him? Is that right?

**Comment by adacovsk** [comment]
> > be changeling
> > spend resources to split and make more changelings
> > they're incentivized to kill me
> > I only vaguely read the first pr but is this right?

More incentive. I made absorb not be able to absorb your own DNA (that would be weird), but Kurfurst wanted to incentivize lings killing eachother instead of being too friendly with eachother. As of right now, I don't really know what to do about that. I'd like to think of it like greytiding an assistant and telling them to blow up the station, rather than directly killing you (as that would be boring and stupid)

**Comment by adacovsk** [comment]
> I only vaguely read the first PR but now there's only one ling when the ruleset fires, he spends resources to spawn another ling from ghosts which are then incentivized to kill him? Is that right?

Perhaps the more I think about it, the less it makes sense. But the idea would just have them to be less "friendly" to eachother. No idea on how to implement it just yet.

**Comment by Eneocho** [comment]
> > I only vaguely read the first PR but now there's only one ling when the ruleset fires, he spends resources to spawn another ling from ghosts which are then incentivized to kill him? Is that right?
> 
> Perhaps the more I think about it, the less it makes sense. But the idea would just have them to be less "friendly" to eachother. No idea on how to implement it just yet.

You could make it so they get a point reward per ling they spawn so they have an incentive to do it, but each ling is independent and can kill other lings to syphon points from (and maybe get some extra points for doing it too why not), making cooperation viable, but betrayal a real tempting option.

**Comment by adacovsk** [comment]
> kill other lings to syphon points from

this already happens, you steal everything from the other ling and leave them as a blank slate

**Comment by Eneocho** [comment]
Then give both a reward for making more lings and for stealing other ling's juice, so there's a reason to make more lings and to betray.

**Comment by adacovsk** [comment]
> Then give both a reward for making more lings and for stealing other ling's juice, so there's a reason to make more lings and to betray.

It's already easy enough to make more lings. I'll maybe add an additional evolve point for absorbing

**Comment by Eneocho** [comment]
That works.

**Comment by gurfan** [comment]
I dislike the change to stasis, but the other things are fine

**Review by gurfan — APPROVED** [review]
didn't mean to start a review

File: code/modules/spells/changeling/regenerative_stasis.dm [hunk 1]
```diff
@@ -5,6 +5,8 @@
 	hud_state = "regenstasis"
 
 	spell_flags = NEEDSHUMAN | STATALLOWED
+	charge_max = 8 MINUTES
+	cooldown_min = 8 MINUTES
```
Comment (gurfan) [inline code/modules/spells/changeling/regenerative_stasis.dm:9]: this should be in the changelog

Comment (adacovsk) [reply code/modules/spells/changeling/regenerative_stasis.dm:9]: it is, but I removed the other constraint on Regenerative Stasis. Please updoot.

**Pull request merged**